src/modules/uniplayer/components/Uniplayer.jsx

```diff
@@ -185,6 +185,7 @@ class Uniplayer extends React.Component {
 							<h5 className="song-title">{currentlyPlaying.info.title}</h5>
 						</div>
 					)}
+					<div className="song-added">Song added!</div>
```

Comment:
Any way we can have this item pop up briefly when a track is added to the playlist?

I'm trying to allow users to be able to click a song from the search and show them that something happened to the greater playlist with this little indicator. 

I also am maintaining the search even once you click an item so you can add multiple songs at a time rather than having to type out the query each time you want to add a track.